models/User.js

```diff
@@ -0,0 +1,26 @@
+var mongoose = require('mongoose');
+
+var UserSchema = new mongoose.Schema({
+  name: String,
+  school: String,
+  day: {
+    type: Number,
+    min : 1,
+    max : 5
+  },
+  phoneNumber: String
+}, {timestamps: true});
+
+// Requires population of author
```

Comment:
What does this comment mean?

Reply:
Whoops I copied the faq schema and left this comment in by accident, not sure what it's referring to